src/api/include/migraphx/migraphx.hpp

```diff
@@ -1255,7 +1341,9 @@ struct experimental_custom_op_base
     virtual std::string name() const                                            = 0;
     virtual argument compute(context ctx, shape output, arguments inputs) const = 0;
     virtual shape compute_shape(shapes inputs) const                            = 0;
-    virtual ~experimental_custom_op_base()                                      = default;
+    virtual std::vector<size_t> output_alias(shapes) const { return {}; }
+    virtual bool runs_on_offload_target() const = 0;
```

Comment:
Can you add a `TODO` comment to return a `string`?

Reply:
Done